.yamato/_abv.yml

```diff
@@ -117,6 +115,120 @@ trunk_verification_trunk:
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+    variables:
+        UPM_REGISTRY: https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates
+    commands:
+      -  curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr.bat --output TestProjects/SRP_SmokeTest/utr.bat
+      -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
+      -  cd TestProjects/SRP_SmokeTest && unity-downloader-cli -u 4ddb31178f9c097b15abff1c38668658c22429b2 -c editor --wait --published-only
+      -  cd TestProjects/SRP_SmokeTest && utr --suite=playmode --testproject=. --editor-location=.Editor --artifacts_path=test-results
+    artifacts:
+        logs:
+            paths:
+              -  "**/test-results/**"
+smoke_test_editmode_2020.2:
+    name: SRP Smoke Test - editmode_2020.2
+    agent:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+    variables:
+        UPM_REGISTRY: https://artifactory-slo.bf.unity3d.com/artifactory/api/npm/upm-candidates
+    commands:
+      -  curl -s https://artifactory.internal.unity3d.com/core-automation/tools/utr-standalone/utr.bat --output TestProjects/SRP_SmokeTest/utr.bat
+      -  pip install unity-downloader-cli --index-url https://artifactory.prd.it.unity3d.com/artifactory/api/pypi/pypi/simple --upgrade
+      -  cd TestProjects/SRP_SmokeTest && unity-downloader-cli -u 4ddb31178f9c097b15abff1c38668658c22429b2 -c editor --wait --published-only
+      -  cd TestProjects/SRP_SmokeTest && utr --suite=editor --platform=editmode --testproject=. --editor-location=.Editor --artifacts_path=test-results
+    artifacts:
+        logs:
+            paths:
+              -  "**/test-results/**"
+all_smoke_tests_2020.2:
+    name: All Smoke Tests - 2020.2
+    dependencies:
+      -  path: .yamato/_abv.yml#smoke_test_playmode_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/_abv.yml#smoke_test_editmode_2020.2
+         rerun: on-new-revision
+all_project_ci_2020.2:
+    name: _ABV for SRP repository - 2020.2
+    dependencies:
+      -  path: .yamato/_projectcontext.yml#all_package_ci_project_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-universal.yml#All_Universal_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-shadergraph.yml#All_ShaderGraph_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-hdrp.yml#All_HDRP_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-vfx_hdrp.yml#All_VFX_HDRP_2020.2
+         rerun: on-new-revision
+      -  path: .yamato/all-vfx_urp.yml#All_VFX_URP_2020.2
+         rerun: on-new-revision
+    triggers:
+        expression: pull_request.target eq "master" AND NOT pull_request.draft AND NOT pull_request.push.changes.all match ["**/*.md", "doc/**/*", "**/Documentation*/**/*", ".github/**/*", "Tools/**/*"]
```

Comment:
we should double check that everyone is okay with their PRs testing against 2020.2 automatically